test_util/aws.py

```diff
@@ -424,6 +424,7 @@ class VpcCfStack(CfStack):
     @classmethod
     def create(cls, stack_name, instance_type, instance_os, instance_count,
                admin_location, key_pair_name, boto_wrapper):
+
```

Comment:
Also, not sure if I understood your comment properly.  If requested for a different PR: https://github.com/dcos/dcos/pull/1249 

And my intention is to squash all the commits into a single One before merging.